nix/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
```

Comment:
Can we document how it was generated or did you just copy it from somewhere (in which case: where)? In case anything needs to be version-bumped in the future.

Reply:
Some part of the file was auto-generated by the `niv` command, it helps in dependency management.
(reference - https://github.com/nmattia/niv).